docs/src/main/tut/applicative.md

```diff
@@ -43,6 +43,6 @@ into the other context:
 Applicative functors are a generalization of Monads thus allowing to express 
 effectful computations into a pure functional way.
 
-Applicative functors are generally preferred to monads when the structure 
+Applicative functors are generally preferred to Monads when the structure 
```

Comment:
I guess my feeling for all of our documentation is that when we are talking about *applicative functors* or *monads* as concepts, we should use lower-case (in that sentence, I think "Applicative functor" starts the sentence so it is OK), and when we are referring to the types in cats we should use `Monad` or `Applicative`.

Does this make sense? If so, feel free to make more invasive changes to standardize on this, or defer those to a future enhancement.